### DISCUSSION TITLE
Suggestion: AnomalibGPT

### DISCUSSION THREAD

**Discussion by monkeycc** [open-edge-platform/anomalib#1351] [Feature Requests]
Discovering an interesting defect detection big language model

https://github.com/CASIA-IVA-Lab/AnomalyGPT

Can Anomalib also create a large language model
llama2

Combined with anomalib

Create a new version

![demo_8](https://github.com/openvinotoolkit/anomalib/assets/6490927/3a2fa8c6-3447-491c-8366-ac234463c0ca)
![demo_5](https://github.com/openvinotoolkit/anomalib/assets/6490927/ff5bec16-e244-4f24-acdc-dbc8f92e0d68)
![demo_1](https://github.com/openvinotoolkit/anomalib/assets/6490927/cd5a341b-616e-4679-8f3e-c296aa427781)

**Comment by samet-akcay** [comment]
@monkeycc, yes we are aware of this repo and keep an eye on it. 

We first would like to implement WinCLIP that was published at CVPR 22. We will then investigate if AnomalyGPT or other LLMs can be integrated into anomalib.

**Comment by monkeycc** [comment]
https://uni-vad.github.io/
@samet-akcay
